blockly-ws-search/test/main.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Example of using WorkspaceSearch with Blockly.
```

Comment:
Is this description correct for this file?

Reply:
I don't think its necessarily incorrect, but this could be updated to "Testing playground for WorkspaceSearch."